src/zmap.c

```diff
@@ -388,11 +389,25 @@ int main(int argc, char *argv[])
 	// other command-line helpers (e.g. probe help)
 	log_debug("zmap", "requested ouput-module: %s", args.output_module_arg);
 
-	// zmap's default behavior is to provide a simple file of the unique IP
-	// addresses that responded successfully.
-	if (!strcmp(args.output_module_arg, "default")) {
-		log_debug("zmap", "no output module provided. will use csv.");
+	// ZMap's default behavior is to provide a simple file of the unique IP
+	// addresses that responded successfully. We only use this simple "default"
+	// mode if none of {output module, output filter, output fields} are set.
+	zconf.default_mode = (!(args.output_module_given || args.output_filter_given || args.output_fields_given));
+	if (zconf.default_mode) {
+		log_info("zmap", "By default, ZMap will output the unique IP addresses "
+				"of hosts that respond successfully (e.g., SYN-ACK packet). This "
+				"is equivalent to running ZMap with the following flags: "
+				"--output-module=csv --output-fields=saddr --output-filter='"
+				"success=1 && repeat=0' --no-header-row. "
+				"If you want all responses, select a different output module or "
```

Comment:
Not so much select a different output module, as "explicitly set an output module", since the output module is still CSV here. I don't think you can set `output-module=default` anymore?